File: C++/Example Statement/Example Switch Case/Clear Screen (Switch Case).cpp
#include <iostream>
#include <conio.h>

using namespace std;

int main() {

    int varchc;

    cout << "Menu :\n" << endl;
    cout << "1. First Example             " << endl;
    cout << "2. Second Example            " << endl;
    cout << "3. Third Example             " << endl;
    cout << "4. Fourth Example	        \n" << endl;
    cout << "Enter your Choice : ";
    cin >> varchc;
    switch (varchc)
    {
    case 1: system("cls"); {
        cout << "Example 1st Succeed" << endl;
        break;
    }
    case 2: system("cls"); {
        cout << "Example 2nd Succeed" << endl;
        break;
    }
    case 3: system("cls"); {
        cout << "Example 3rd Succeed" << endl;
        break;
    }
    case 4: system("cls"); {
        cout << "Example 4th Succeed" << endl;
        break;
    }
    default: system("cls"); {
        cout << "Enter your Choice Correctly" << endl;
        break;
    }
    }_getch();

    return 0;

}

File: C#/Example Statement/Example Switch Case/Clear Screen (Switch Case).cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            int varchc;

            Console.WriteLine("Menu :\n");
            Console.WriteLine("1. First Example            ");
            Console.WriteLine("2. Second Example           ");
            Console.WriteLine("3. Third Example            ");
            Console.WriteLine("4. Fourth Example         \n");
            Console.Write("Enter your Choice : ");
            varchc = int.Parse(Console.ReadLine());

            switch (varchc)
            {
                case 1:
                    Console.Clear();
                    Console.WriteLine("Example 1st Succeed");
                    break;
                case 2:
                    Console.Clear();
                    Console.WriteLine("Example 2nd Succeed");
                    break;
                case 3:
                    Console.Clear();
                    Console.WriteLine("Example 3rd Succeed");
                    break;
                case 4:
                    Console.Clear();
                    Console.WriteLine("Example 4th Succeed");
                    break;
                default:
                    Console.Clear();
                    Console.WriteLine("Enter your Choice Correctly");
                    break;
            }

        }
    }
}
